engineering_tools/monitor-ros-cpu.py

```diff
@@ -7,6 +7,37 @@
 import argparse
 from datetime import datetime
 
+# Define ROS-related keywords to filter processes
+ROS_KEYWORDS = {
+    "ros",
+    "node",
+    "rviz",
+    "rqt",
+    "ros2 launch",
```

Comment:
Nit: Shouldn't need `ros2 launch` since `ros2` is already a keyword

Reply:
Makes sense removed